From 5d697053459e693acc8d31e0f91b47758e9969e4 Mon Sep 17 00:00:00 2001
From: Yehuda Yitschak <yehuday@marvell.com>
Date: Tue, 21 Nov 2017 13:42:44 +0200
Subject: [PATCH 2082/2241] dts: a8040-ocp: change 2nd 10G port to SGMII

The 2nd 10g port in the OCP card is not used for data forwarding.
Instead we need to use it as a management port. This patch switches
the 2nd port to SGMII mode to be used with 1g networks

Change-Id: I8e35384bebaa3de259a0216fd926c0f983be2cf1
Signed-off-by: Yehuda Yitschak <yehuday@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/46576
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-ocp.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-ocp.dts b/arch/arm64/boot/dts/marvell/armada-8040-ocp.dts
index 356f733..3523ddc 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-ocp.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-ocp.dts
@@ -172,9 +172,9 @@
 			};
 			gop {
 				cp1_emac0: mac0 {
-					phys = <&cp1_comphy 4 COMPHY_SFI>;
+					phys = <&cp1_comphy 4 COMPHY_SGMII0>;
 					phy-names = "comphy";
-					phy-mode = "sfi"; /* lane-4 */
+					phy-mode = "sgmii"; /* lane-4 */
 				};
 				cp1_emac2: mac2 {
 					phy-mode = "none";
-- 
2.7.4

